source/fem-dem/ib_particles_dem.cc

```diff
@@ -67,6 +67,69 @@ IBParticlesDEM<dim>::update_particles(
   cfd_time      = time;
 }
 
+template <int dim>
+void
+IBParticlesDEM<dim>::update_contact_candidates()
+{
+  particles_contact_candidates.resize(dem_particles.size());
+
+  double radius_factor = parameters->contact_search_radius_factor;
+
+  // If contact_search_radius_factor < 1, all particles are taken into
+  // account in the contact search. Otherwise, do it for the particles inside
+  // the region of contact search. By default, radius_factor is zero, so all
+  // particles are taken into account.
+  if (radius_factor <= 1)
+    {
+      for (auto &particle_one : dem_particles)
+        {
+          for (auto &particle_two : dem_particles)
+            {
+              if (particle_one.particle_id != particle_two.particle_id and
+                  particle_one.particle_id < particle_two.particle_id)
+                {
+                  const Point<dim> particle_one_location =
+                    particle_one.position;
+                  const Point<dim> particle_two_location =
+                    particle_two.position;
+
+                  particles_contact_candidates[particle_one.particle_id].insert(
+                    particle_two.particle_id);
+                }
+            }
+        }
+    }
+
+
+  else
```

Comment:
This whole if, else can be completely removed by just setting a large value of the radius factor if you want everyone to see evryone